package.json

```diff
@@ -52,10 +132,25 @@
                 "category": "rock",
                 "command": "rock.selectPackage"
             },
+            {
+                "title": "Select package type",
```

Comment:
Package types are listed in the installation manifest.

Reply:
You probably noticed it already... This is supposed to be used as a fallback when the package is not registered in the manifest